px-vis-register.html

```diff
@@ -199,6 +199,7 @@
                       disable-click="[[disableClick]]"
                       muted-series="{{mutedSeries}}"
                       number-format="[[numberFormat]]"
+                      register-format-callback="{{registerFormatCallback}}"
```

Comment:
Is this meant to be a two-way binding? If not,
```suggestion
                      register-format-callback="[[registerFormatCallback]]"
```